tests/terminator_tests/test_erroreval.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import math
+
+import pytest
+
+from optuna.study.study import create_study
+from optuna.terminator.erroreval import _CROSS_VALIDATION_SCORES_KEY
+from optuna.terminator.erroreval import CrossValidationErrorEvaluator
+from optuna.terminator.erroreval import report_cross_validation_scores
+from optuna.terminator.erroreval import StaticErrorEvaluator
+from optuna.trial import create_trial
+from optuna.trial import FrozenTrial
+
+
+def _create_trial(value: float, cv_scores: list[float]) -> FrozenTrial:
+    return create_trial(
+        params={},
+        distributions={},
+        value=value,
+        system_attrs={_CROSS_VALIDATION_SCORES_KEY: cv_scores},
+    )
+
+
+def test_cross_validation_evaluator() -> None:
+    study = create_study(direction="minimize")
+    study.add_trials(
+        [
+            _create_trial(value=2.0, cv_scores=[1.0, -1.0]),  # second best trial with 1.0 var
+            _create_trial(value=1.0, cv_scores=[2.0, -2.0]),  # best trial with 4.0 var
```

Comment:
Nit. Sorry I haven't checked the line length.
```suggestion
            _create_trial(value=2.0, cv_scores=[1.0, -1.0]),  # Second best trial with 1.0 var.
            _create_trial(value=1.0, cv_scores=[2.0, -2.0]),  # Best trial with 4.0 var.
```